tests/test_units/test_utils.py

```diff
@@ -85,42 +84,6 @@ def test_get_object_name():
     )
 
 
-@pytest.mark.usefixtures("inside_rms_interactive")
```

Comment:
the tests here were just moved since the functionality have been moved to the `_runcontext.py`